include/openssl/buffererr.h

```diff
@@ -8,18 +8,15 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef OPENSSL_BUFFERERR_H
-# define OPENSSL_BUFFERERR_H
+#ifndef OPENSSL_BUFERR_H
+# define OPENSSL_BUFERR_H
```

Comment:
Why this rename?

Reply:
It took me a moment, but yeahofcourse, the guard macro name is based on the sub-library name with ERR appended, not on the file name, and since the sub-libbrary name is `BUF`...

I'll switch to using the header file name as a template instead, that makes more sense

Reply:
Fixed.  This lead to other renames, but they are at least easier to understand, since the renames match the header files.